### ISSUE TITLE
Doc errors

### ISSUE THREAD

**Issue by alecloudenback** [JuliaComputing/Miletus.jl#11]
The docs seem to have an issue building/rendering the gadfly plots. I tested locally and don't see the same issue building the docs.

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/711879/194459519-80f0f97b-e456-4aa7-8a0e-2b5f32336ae9.png">

**Comment by alecloudenback** [comment]
Looks like it's fixed now

**Comment by aviks** [comment]
Yes, apologies, I should have responded here -- I did a new release with the updated docs a couple of weeks ago.
